benchmark_runner/common/template_operations/templates/hammerdb/internal_data/mariadb_template.yaml

```diff
@@ -113,7 +113,8 @@ spec:
       terminationGracePeriodSeconds: 10
       containers:
       - name: mariadb
-        image: docker.io/centos/mariadb-103-centos8:latest
+        # docker.io/centos/mariadb-103-centos8:latest - rate limit issue
+        image: quay.io/ebattat/mariadb-103-centos8
```

Comment:
@RobertKrawitz, we hit the following error for docker.io images when running it from Perf Labs:
`podman WARN[0063] Failed, retrying in 1s ... (2/3). Error: initializing source docker://centos/postgresql-10-centos8:latest: reading manifest latest in docker.io/centos/postgresql-10-centos8: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit`
Thats the reason I move all docker.io images to quay.io.